### PR TITLE
Edit Existing Experience

### LINKED ISSUE #5: Page for editing existing Experience
https://github.com/MLH-Fellowship/orientation-project-python-23.fal/issues/15

### PR DESCRIPTION
**Pull request by jescalada**
This is a PR for #5 (which I implemented accidentally instead of my assigned issue #3). It adds an Edit button which opens a form that lets you edit and save the new values to the state.

image

Known Issue: This is not making any calls to the backend, it is only modifying the frontend state!

### CODE REVIEW

**Review by Pradyuman7 — COMMENTED** [review]
Based on the screenshot, can we have some spacing between the inputs? It seems a bit compact right now

**Comment by jescalada** [comment]
Got it! I just made it so that the sections as well as the edit form are a bit more responsive (it's not perfect, but now it's usable regardless of screen size).

File: src/components/ExperienceContainer/ExperienceContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    const description = document.getElementById("description").value;
+    const logo = document.getElementById("logo").value;
+
+    // Delete the content of the experience card
+    experienceCard.innerHTML = "";
+
+    // Append new content
+    const html = `<div class="content">
+                    <h3>
+                      ${company} - ${title}
```
Comment (Pradyuman7) [inline src/components/ExperienceContainer/ExperienceContainer.jsx:32]: nit: setting inner html usually makes it easy for the XSS attacks, but it's not really part of this issue, so please use your best judgement about this. If you think you can fix it, go for it, if not, maybe we can track it as a separate issue and fix it for that.

**Review by Pradyuman7 — DISMISSED** [review]
Approving with the comment, please take a look at that

File: src/components/ExperienceContainer/ExperienceContainer.jsx (hunk 1 above)
Comment (jescalada) [reply src/components/ExperienceContainer/ExperienceContainer.jsx:32]: Would it be okay if I used an HTML sanitization library to parse the HTML (while keeping the innerHtml call)? It seems it's a [valid solution](https://coderpad.io/blog/development/javascript-innerhtml/):

**Comment by jescalada** [comment]
Resolved a merge conflict. Pinging @Pradyuman7 for a quick review :)

**Review by Pradyuman7 — APPROVED** [review]

**Pull request merged**